articles/iot-central/core/howto-transform-data.md

```diff
@@ -156,7 +156,7 @@ To create a device template for the IoT Edge gateway device:
 
 1. Open your local copy of the *moduledeployment.json* manifest file in a text editor.
 
-1. Find the `registryCredentials` section and replace the placeholders with the values you made a note of when you created the Azure container registry. The `address` value looks like `{your username}.azurecr.io`.
+1. Find the `registryCredentials` section and replace the placeholders with the values you made a note of when you created the Azure Container Registry. The `address` value looks like `{your username}.azurecr.io`.
```

Comment:
This one should be left as is. When we refer to an instance we use lower case. Uppercase is reserved for when we refer to the service. Take a look at https://learn.microsoft.com/en-us/azure/container-registry/container-registry-intro for examples.